### ISSUE TITLE
How to change hop length or upscale_factors

### ISSUE THREAD

**Issue by dhgrs** [dhgrs/pytorch-UniWaveNet#2]
@switchzts
I watched your repository forked from mine and noticed that you changed `hop` in `params.py`. And you should change `upscale_factors` to the product of them equals to `hop`. And `length` in `params.py` must be multiple of `hop`.
Because `hop` is hop length of STFT to generate mel spectrogram for conditioning and `upscale_factors` is deconvolution's strides for upscaling mel spectrogram into wave form scale.

**Comment by switchzts** [comment]
Ok, you are so nice. Because my sampling rate is 16000, so I set the hop to 200 (it seems to be better if set it to around fram_sift(12.5ms)*sampling rate according to Google's research)。Then I started running the error. When I actually ran it, I set it back to 256. It turned out to be the dimension problem of unsapple. This should not affect much(200 or 256).I have updated my code.
BTW, in the original text, there is some phase loss added to the loss. Do you have any thoughts of it?

**Comment by switchzts** [comment]
Hey, u r so nice. Thanks. I first train the code and get an error about dimensions,so I turn it back to 256.I change it because my sampling rate was 16000.
BTW, in the paper they said that add phase loss to model will help model get better audio.Do you have any thought of it?

**Comment by switchzts** [comment]
?

**Comment by dhgrs** [comment]
I don't mind creating many issues so can you open a new issue for a new topic?
Then other people can find issues they looking for easily.
